tests/integration/examples/example_tests/tcp_listener_fd_passing.rs

```diff
@@ -0,0 +1,50 @@
+use super::utils;
+use rama::http::BodyExtractExt;
+use serde_json::Value;
+
+#[tokio::test]
+#[ignore]
+async fn test_tcp_listener_fd_passing() {
+    utils::init_tracing();
+
+    let runner = utils::ExampleRunner::interactive("tcp_listener_fd_passing", None);
+
+    // Retry until child process is serving (FD handoff takes variable time)
+    let response = async {
+        for i in 0..8 {
+            match runner.get("http://127.0.0.1:62046").send().await {
```

Comment:
The runner already has a retry layer applied to to it (https://github.com/plabayo/rama/blob/fa0f41e68d7e06fe116565da808ab4810aaa5516/tests/integration/examples/example_tests/utils/mod.rs#L188-L198), did it not work for you without this manual retry logic?